test/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDaoTest.java

```diff
@@ -475,6 +477,36 @@ public void getAllSurveysMostRecentVersion() throws Exception {
     }
 
     @Test
+    public void canGetSummarySurvey() throws Exception {
+        // First, it returns no responses without error
+        List<Survey> surveys = surveyDao.getSurveysSummary(studyIdentifier);
+        assertEquals(0, surveys.size());
+        
+        // Add an information screen so we can verify it is removed.
+        DynamoSurveyInfoScreen info = new DynamoSurveyInfoScreen();
+        info.setIdentifier("info");
+        info.setPrompt("This is the prompt.");
+        info.setPromptDetail("This is the prompt detail.");
+        info.setTitle("This is a title.");
+        
+        // Create and publish a couple of surveys
+        Survey survey = new TestSurvey(true);
+        survey.getElements().add(info);
+        survey = createSurvey(survey);
+        survey = publishSurvey(studyIdentifier, survey);
+        
+        survey = new TestSurvey(true);
+        survey.getElements().add(info);
+        survey = createSurvey(survey);
+        survey = publishSurvey(studyIdentifier, survey);
+        
+        surveys = surveyDao.getSurveysSummary(studyIdentifier);
+        assertEquals(2, surveys.size());
```

Comment:
Similarly, you rely on there being exactly 2 surveys, which also won't always be true. It's better to check that the results contain the surveys you expect and/or that all surveys return don't have survey info screens.